spec/hardware/iommu.fmf

```diff
@@ -0,0 +1,31 @@
+summary: |
+    Select or provision a guest with the `Input–output memory management unit`.
+
+description: |
+    .. code-block::
+
+       iommu:
+           # Boolean, whether the guest supports IOMMU
+           is-supported: true|false
+           # String, the model name for IOMMU
+           model: "virtio"|"intel"|"smmuv3"|...
+
+    .. versionadded:: 1.36
+
+    .. versionchanged:: 1.36
+       ``beaker`` plugin supports ``iommu``
+
+example:
+  - |
+    # Require a guest that supports IOMMU
+    iommu:
+        is-supported: true
+
+  - |
+    # Require a guest with the IOMMU of virtio model
+    iommu:
+        is-supported: true
+        model: virtio
+
+link:
+    - implemented-by: /tmt/steps/provision/mrack.py
```

Comment:
IIUIC, the patch adds support for `is-supported` field only to Beaker plugin, please add a similar note here: https://github.com/teemtee/tmt/blob/main/spec/hardware/system.fmf#L37